pages/api/dynamodb-client-helper.ts

```diff
@@ -0,0 +1,15 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
+
+/**
+ * Helper function to instantiate DynamoDB and Document client.
+ * https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/dynamodb-example-document-client.html
+ */
+export function connectToDynamo(): DynamoDBDocumentClient {
```

Comment:
This file shouldn't be under the `/pages/api` folder structure.  NextJS uses folder structure to generate API endpoints and will actually create an api endpoint at the path `api/dynamodb-client-helper`.

Reply:
Oops! Will change it